src/pybind/CMakeLists.txt

```diff
@@ -1,33 +1,42 @@
-find_package(Cython REQUIRED)
 include(Distutils)
 
 set(CYTHON_MODULE_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cython_modules)
 
-add_subdirectory(rados)
-add_subdirectory(rbd)
-add_subdirectory(cephfs)
-
-add_custom_target(cython_modules ALL
-  DEPENDS cython_rados cython_cephfs cython_rbd)
-
-# if CMAKE_INSTALL_PREFIX is an empty string, must replace
-# it with "/" to make PYTHON_INSTALL_TEMPLATE an absolute path to be
-# consistent with all other installation paths.
-if(CMAKE_INSTALL_PREFIX)
-  set(PYTHON_INSTALL_TEMPLATE "${CMAKE_INSTALL_PREFIX}")
-else(CMAKE_INSTALL_PREFIX)
-  set(PYTHON_INSTALL_TEMPLATE "/")
-endif(CMAKE_INSTALL_PREFIX)
-
-execute_process(
-  COMMAND
-  ${PYTHON_EXECUTABLE} -c "from distutils import sysconfig; print sysconfig.get_python_lib(prefix='${PYTHON_INSTALL_TEMPLATE}')"
-  OUTPUT_VARIABLE PYTHON_INSTDIR
-  OUTPUT_STRIP_TRAILING_WHITESPACE)
-
-install(FILES
-  ${CMAKE_CURRENT_SOURCE_DIR}/ceph_argparse.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/ceph_daemon.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/ceph_rest_api.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/ceph_volume_client.py
-  DESTINATION ${PYTHON_INSTDIR})
+foreach(python_version "" "3")
+  set(PYTHON_VERSION ${python_version})
+
+  find_package(Python${PYTHON_VERSION}Interp ${PYTHON_VERSION} QUIET)
```

Comment:
just want to understand why the stock `FindPythonInterp.cmake` does not work in this case. as it lists "3.4 3.3 3.2 3.1 3.0" as supported python3 versions (in cmake 3.3.0). maybe the `FindPythonInterp.cmake` packaged by the supported distros is too old?